blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-drag.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-drag.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-drag.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+## Animations
+
+The predefined ialog can be animated during the open and close actions. Also, users can customize animation’s [DialogAnimationSettings.Delay](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Delay), [DialogAnimationSettings.Duration](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Duration) and [DialogAnimationSettings.Effect](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Effect) by using the `DialogAnimationSettings` property.
+
+In the following sample, `Zoom` effect is enabled. So, The Dialog will open with `ZoomIn` and close with `ZoomOut` effects.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-animation.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+## Position
+
+You can customize the dialog position by using [Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.Position.html#fields) property. Use the following code to customize the dialog position.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-position.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert position Dialog](./images/blazor-alert-position.png)
+
+![Confirm position Dialog](./images/blazor-confirm-position.png)
+
+![Prompt position Dialog](./images/blazor-prompt-position.png)
+
+## Dimension
+
+You can customize the dialog dimensions using [DialogDimension.Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogDimension.html#Syncfusion_Blazor_Popups_DialogDimension_Height) and [DialogDimension.Width] properties. Use the following code to customize the dialog dimensions.
```

Comment:
You can customize the dialog dimensions using [DialogDimension.Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogDimension.html#Syncfusion_Blazor_Popups_DialogDimension_Height) and [DialogDimension.Width] properties. Use the following code to customize the dialog dimensions: